src/connectionController.ts

```diff
@@ -304,13 +305,6 @@ export default class ConnectionController {
 
     const connectionStringData = new ConnectionString(connectionString);
 
-    // TODO: Allow overriding appname + use driverInfo instead
-    // (https://jira.mongodb.org/browse/MONGOSH-1015)
-    connectionStringData.searchParams.set(
```

Comment:
so this is problematic because we have been storing appended app name URLs which would mean they're reporting a vscode version from the time that the connection was created... and also not append this extra info for the new users.

my quick solution is to adjust each time but it'd be good to do an adjustment to the saved connection URL itself. look a bit more into that option